lib/include/public/IOfflineStorage.hpp

```diff
@@ -33,12 +33,13 @@ namespace MAT_NS_BEGIN {
         StorageBlob     blob;
         int             retryCount = 0;
         int64_t         reservedUntil = 0;
+        std::string     traceId;
```

Comment:
Can we have this definition under macro, this will ensure that `StorageRecord` size will not change for non windows platform, if we plan to merge these changes to main.

Reply:
done